### ISSUE TITLE
Questions about the training strategy？

### ISSUE THREAD

**Issue by feiyuhuahuo** [dbolya/yolact#185]
I'm training yolact with a different backbone. But the convergence is really slow. I don't know whether this backbone works well. I have three questions:
1. Can the learning rate always be same for different backbones?
Noticed you tried resnet101, resnet50, darknet53, and SSD uses vgg16. They both use learning rate=0.001. So 0.001 is OK with many kinds of backbone?

2. As far as I know, the momentums in SGD for PyTorch and Caffe are different. SSD are written by Caffe with momentum=0.9,  and Yolact are written by PyTorch. The momentum in Caffe actually equals to learning_rate * momentum in PyTorch. So the momentum in your code makes a little effect？ If enlarges the momentum, the training process would be more stable?

3. When tuning parameters, did you firstly experiment on VOC and then test on COCO? If a strategy improves the performance on VOC, does this  strategy also works for COCO? By the way, could VOC use for training masks, I really don't know. :)
Thanks for the reply.

**Comment by dbolya** [comment]
1. I'd imagine that depends on how different the backbones are. `1e-3` is just the typical convention for convnets, but it might not be optimum for either resnet or VGG, it's just what people use. I know, for instance, that the [optimum lr also depends on the batch size](https://arxiv.org/abs/1706.02677). `1e-3` should be a good starting point though.
2. I actually didn't know this, thanks for point it out. According to [this post](https://discuss.pytorch.org/t/sgd-compatibility-with-other-frameworks/19960/11), the two methods should be equivalent unless the learning rate changes, so it might not have too much of an effect. I'll take a look at how mask r-cnn's pytorch repo deals with momentum and do some experiments later, but that'll have to wait until after ICCV.
3. Nah, we just went full power on COCO. VOC does have SBD, which has mask annotations, but I didn't actually have that dataset implemented until recently. I'll be pushing it to the master branch soon, so if you want to try checking your experiments on VOC first, you can.

**Comment by feiyuhuahuo** [comment]
I afterwards knows the mechanism of the momentum difference. And thank for the replies for other two questions.
